.gitignore

```diff
@@ -0,0 +1,7 @@
+characters_counter/__pycache__/
```

Comment:
Generic .gitignore - https://github.com/github/gitignore/blob/main/Python.gitignore
Ctrl C + Ctrl V
And no need to worry about anything new

Reply:
Ok, it is easy =)